core-services/release-controller/_repos/ocp-4.16-rhel94.repo

```diff
@@ -36,7 +36,7 @@ failovermethod = priority
 
 [rhel-9.4-nfv]
 name = rhel-9.4-nfv
-baseurl = https://cdn.redhat.com/content/beta/rhel9/9/x86_64/nfv/os
+baseurl = https://cdn.redhat.com/content/e4s/rhel9/9.4/x86_64/nfv/os/
```

Comment:
Not sure what `e4s` is, but it differs from the `eus` above... hence this comment.

Reply:
The NFV repos are in a different place as they have a different lifecycle. See for RHEL 9.2: https://github.com/openshift/release/blob/master/core-services/release-controller/_repos/ocp-4.15-rhel92.repo#L39

Reply:
There are some hints in https://access.redhat.com/articles/3751271

Reply:
That explains it, thanks.
/lgtm